canonical-web-and-design/snapcraft.io
# Command consistency

Fixes [#2048](https://github.com/canonical-web-and-design/snapcraft.io/issues/2048)

## QA Steps
1. Pull the branch or run the demo service
2. Check out all language guides include colons (:) before a command
3. Rinse and repeat at least 8 times
4. #watchhowmetricsgoup #winning
